source/configuration_manual/push_notification.rst

```diff
@@ -494,3 +496,92 @@ Example with event code:
        e:log_info("Mailbox notify status " .. tostring(code))
      end
    end
+
+Chronos driver [``chronos``]
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to the :ref:`push_notification_ox` the Chronos backend supports sending
+notifications on MessageNew events in case the message contains a calendar
+invite.
+
+This driver was designed for use with the `OX App Suite iCalendar
+Transport-Independent Interoperability Protocol (iTIP)
+<https://documentation.open-xchange.com/7.10.6/middleware/calendar/iTip.html#rest-api>`_,
+but can be used by any endpoint that implements the same API, not just OX App
+Suite.
```

Comment:
Note for reviewer: Maybe I should add a paragraph that outlines the fact that this is not yet publicly available?

Reply:
Shouldn't .. versionadded be here, and enough for that?

Reply:
I can add this as well, but I specifically thought about the "public" part.